### ISSUE TITLE
how to run this application

### ISSUE THREAD

**Issue by ravigithubshankar** [SEmohamedAhmed/LSD-VTON#1]
guide me to how to run this application?

**Comment by SEmohamedAhmed** [comment]
Please, follow the repository for more updates we will add what you want soon.
@ravigithubshankar

**Comment by ravigithubshankar** [comment]
yeah iam following the repo but how to run in our laptops there is no
instructions can pls provide that instructions its very urjent..

**Comment by ravigithubshankar** [comment]
ur repo is seems to be everything is fine ,but when the other developers also want to explore ur repo then without proper guidelines how can it possible?

**Comment by SEmohamedAhmed** [comment]
I believe there are a lot of errors could be faced while running the repo, I will add the guidelines soon.

**Comment by ravigithubshankar** [comment]
actually we are using ur repo for project purpose so that we are requesting
to pls send me the guidelines to my personal email :
***@***.***

**Comment by ravigithubshankar** [comment]
actually we are using ur repo for project purpose so that we are requesting to pls send me the guidelines to my personal email : [ravishankarreddybedadhala@gmail.com](mailto:ravishankarreddybedadhala@gmail.com)
